IceCubesApp/App/Report/ReportView.swift

```diff
@@ -44,6 +44,7 @@ public struct ReportView: View {
           ToolbarItem(placement: .navigationBarTrailing) {
             Button {
               isSendingReport = true
+              let client = client
```

Comment:
May i ask why this was needed?

Reply:
I think it was one of the beta bug, I'll try to remove it.

Reply:
Removed